pkg/fakeclockgettime/fake_clock_gettime.c

```diff
@@ -0,0 +1,25 @@
+#include <time.h>
```

Comment:
This file was just used to show the logic of these byte code. (We cannot use `dlopen` to open a so image now, so we cannot compile these codes and just use them.)

Reply:
Did the byte code has compatibility issues during Macos and linux

Reply:
Yes. These byte codes can only run on linux 64-bit x86 platform.

As Mac OS programs use a different way to get time, we will not support time chaos on Mac OS. But if needed, we can support more cpu architecture in the future.

Reply:
LGTM. then maybe we should add a `//TODO` in code

Reply:
Soon. we need to support ARM. so I think later we should support other cpu architecture too.

Reply:
OK. I add a `//TODO` right before `var fakeImage = []byte{`